models/reset_password/reset_password.go

```diff
@@ -39,7 +39,7 @@ func GenerateResetPassword(userID int64, email string) *ResetPasswordStruct {
 	return NewResetPassword(0, userID, token, time.Now().AddDate(0, 0, 1))
 }
 
-func Authenticate(id int64, token string) bool {
+func Authenticate(id int64, token string) error {
```

Comment:
__[golint]__
* [golint] exported function Authenticate should have comment or be unexported